File: src/main/java/com/shopping/shopping_site_backend/infra/constant/enumconstant/PaymentStatus.java
package com.shopping.shopping_site_backend.infra.constant.enumconstant;

public enum PaymentStatus {
  PENDING("Pending"), // 待支付
  COMPLETED("Completed"), // 已完成
  FAILED("Failed"), // 失败
  REFUNDED("Refunded");
  private String description;

  PaymentStatus(String description) {
    this.description = description;
  }
}

File: src/main/java/com/shopping/shopping_site_backend/infra/constant/enum/PaymentStatus.kt
package com.shopping.shopping_site_backend.infra.constant.enum

enum class PaymentStatus(val status: String) {
    PENDING("Pending"),  // 待支付
    COMPLETED("Completed"),  // 已完成
    FAILED("Failed"),  // 失败
    REFUNDED("Refunded") // 已退款
}
